### DISCUSSION TITLE
Legibility and updates suggestion

### DISCUSSION THREAD

**Discussion by quadransmuralis** [SpiralLinux/SpiralLinux-project#116] [Q&A]
I'm running Spiral Linux gnome: would it be possible to make the UI fonts black rather than grey and so make the UI more legible?

A second suggestion concerns updates: would it be possible to install them when one shuts down the computer, which by definition is when one doesn't need the machine anymore, rather than on booting it up, which by definition is when one needs one's machine? Psychologically it would be less frustrating/irritating.

Otherwise, it works flawlessly for me on a DELL aio inspiron 22 3000 i5.

**Comment by geckolinux** [comment]
Hi there, glad to hear it's generally working well for you, and thanks for the question.

> would it be possible to make the UI fonts black rather than grey and so make the UI more legible?

I agree this would be ideal. It is definitely possible technically, in fact I personally use a custom GTK theme with pure black text. But I prefer to only ship SpiralLinux with themes that are available in the Debian repos, and unfortunately there are none with pure black text. The default Numix theme is the best one in the repos, but it does leave something to be desired. I would recommend you take a look at the oomox project to fairly easily generate your own custom GTK theme:
https://github.com/themix-project/oomox

> A second suggestion concerns updates: would it be possible to install them when one shuts down the computer, which by definition is when one doesn't need the machine anymore, rather than on booting it up

I assume that you're using Gnome Software for updating the system. I actually don't like or recommend Gnome Software for system updates and/or for managing native DEB packages. Its primary function in SpiralLinux is as a graphical manager for Flatpaks. Please take a look at the wiki recommendation for updating and/or upgrading the base Debian system and apps with Synaptic:
https://github.com/SpiralLinux/SpiralLinux-project/wiki#updating-and-upgrading-spirallinux

**Reply by quadransmuralis** [reply]
Currently I'm using LMDE 5. I used to play with Linux but nowadays I just want to get on with my work, which means I'm always on the lookout for a distro that does what it says.

SpiralLinux seems to fall into that category (so many distros really don't). So, BRAVO!

But on top of that, I want a really legible distro!

**Reply by geckolinux** [reply]
Hi there, I agree completely, and I'm in the same category. I just need to get work done during long periods in front of the computer, and to do so I need the best quality font rendering.

The last time I tried LMDE it had poor font color contrast with its default theme. But if it does come with one that has better contrast you can copy that theme from `/usr/share/themes/` and use it in any other distro such as SpiralLinux.

Another option instead of creating an entire theme with oomox is to follow this tutorial to change just the font color to black on an existing theme:
https://www.dedoimedo.com/computers/ubuntu-mate-theme-edit-fonts-black.html

Finally, if you get along with Plasma you could use the SpiralLinux Plasma edition, which does have pure black fonts out of the box thanks to the color configuration options in Plasma.

**Reply by quadransmuralis** [reply]
SpiralLinux Plasma might be the alternative I'm looking for! I shall spend some time with it.
Thx.
